doc/send_request.md

```diff
@@ -0,0 +1,269 @@
+# Using your Client's `send_request` Method
+
+In this doc, we will be showing how to build your own ***http requests*** and how to send these requests directly to the service using the ***send_request*** method.
+
+Here's how to get started:
+
+```python
+>>> from azure.identity import DefaultAzureCredential
+>>> from azure.example.service import ExampleClient
+>>> from azure.core.rest import HttpRequest, HttpResponse
+>>> client = ExampleClient(endpoint="https://www.example.org/", credential=DefaultAzureCredential())
```

Comment:
Should we have a note for clarity "client creation may vary for your SDK, please refer to your SDK doc for this part"